internal/client/ca_certificate.go

```diff
@@ -0,0 +1,50 @@
+package infisicalclient
+
+import (
+	"fmt"
+	"terraform-provider-infisical/internal/errors"
+)
+
+const (
+	operationGenerateCACertificate = "CallGenerateCACertificate"
+	operationGetCACertificate      = "CallGetCACertificate"
+)
+
+func (client Client) GenerateCACertificate(request GenerateCACertificateRequest) (GenerateCACertificateResponse, error) {
+	var certResponse GenerateCACertificateResponse
+	response, err := client.Config.HttpClient.
+		R().
+		SetResult(&certResponse).
+		SetHeader("User-Agent", USER_AGENT).
+		SetBody(request).
+		Post(fmt.Sprintf("api/v1/cert-manager/ca/internal/%s/certificate", request.CaId))
+
+	if err != nil {
+		return GenerateCACertificateResponse{}, errors.NewGenericRequestError(operationGenerateCACertificate, err)
+	}
+
+	if response.IsError() {
+		return GenerateCACertificateResponse{}, errors.NewAPIErrorWithResponse(operationGenerateCACertificate, response, nil)
+	}
+
+	return certResponse, nil
+}
+
```

Comment:
IGNORE PLACEMENT OF COMMENT

I defined the parent CA ID in Terraform but it doesn't look like it got linked behind the scenes? 

<img width="1205" height="951" alt="image" src="https://github.com/user-attachments/assets/f3a740f0-c091-4e35-8898-9ae2bccc2dff" />
<img width="689" height="468" alt="image" src="https://github.com/user-attachments/assets/f68f7cfe-d864-49a4-9b80-46f66d1a7ed9" />
<img width="712" height="321" alt="image" src="https://github.com/user-attachments/assets/2f67f8cc-2d43-4648-9ea4-1abbe48ba2b8" />